Makefile

```diff
@@ -36,6 +38,17 @@ binary: generate-webui build ## build the linux binary
 crossbinary: generate-webui build ## cross build the non-linux binaries
 	$(DOCKER_RUN_TRAEFIK) ./script/make.sh generate crossbinary
 
+crossbinary-parallel:
+	$(MAKE) generate-webui
+	$(MAKE) build
+	$(MAKE) crossbinary-default crossbinary-others
+
+crossbinary-default:
```

Comment:
Shouldn't this target (and `-others`) depend on `binary` too in case we want to run this stand-alone (i.e., without running `crossbinary-parallel` first)?